content/src/helpers/FetcherFactory.ts

```diff
@@ -13,4 +13,8 @@ export class FetcherFactory {
       }
     })
   }
+
+  static copy(originalFetcher: Fetcher): Fetcher {
+    return new Fetcher(originalFetcher['customDefaults'])
```

Comment:
Does this work? hahah

Wouldn't it be better if `Fetcher` has a `.clone()` or exposed the default config?

Reply:
It works, any way you are right 😂